tasks/config.yml

```diff
@@ -238,6 +238,13 @@
   - restart freeswitch
   - reload systemd
 
+- name: configure IP {{ bbb_freeswitch_ip_address }} for wss-binding of FreeSWITCH
+  lineinfile:
+    path: /opt/freeswitch/conf/sip_profiles/external.xml
+    regexp: "<param name=\"wss-binding\"  value=\":7443\"/>"
```

Comment:
Please change this regex, because re-running this task could lead to the line beeing duplicated in the config.

Reply:
I was already re-running the task when testing and it never duplicated. Since the regex doesn't match anymore once `{{ 
bbb_freeswitch_ip_address }}` gets added, I don't see how that can be a problem? Even when you set  `{{ 
bbb_freeswitch_ip_address }}` to "", it will only replace the line with itself.